pkg/controller/clusterinstallation/mattermost_test.go

```diff
@@ -74,7 +74,10 @@ func TestCheckMattermost(t *testing.T) {
 		require.NoError(t, err)
 		err = r.client.Get(context.TODO(), types.NamespacedName{Name: ciName, Namespace: ciNamespace}, found)
 		require.NoError(t, err)
-		assert.Equal(t, original, found)
+		assert.Equal(t, original.GetName(), found.GetName())
+		assert.Equal(t, original.GetNamespace(), found.GetNamespace())
+		assert.Equal(t, original.Spec.Selector, found.Spec.Selector)
+		assert.Equal(t, original.Spec.Ports, found.Spec.Ports)
 	})
```

Comment:
Are we not able to just compare the whole thing like we used to?

Reply:
no, there are some fields that change for example the `ResourceVersion`